galaxyproject/galaxy
# Render Useful Markdown Components for Mapped Over Steps

Builds on the markdown enhancements in https://github.com/galaxyproject/galaxy/pull/17105. Fixes https://github.com/galaxyproject/galaxy/issues/16863 for job parameters and job metrics in the GUI. ~I want to modernize the other two job targets (tool standard error and standard out) before making it apply to them as well.~ (Edit: done)

## How to test the changes?
(Select all options that apply)
  1. Create a workflow with a mapped over step, give the step a label, add job metrics or job parameters component to the markdown report for the workflow, run the workflow over a small collection, render the report, see the new job selector.

## License
